index.html

```diff
@@ -8,46 +8,19 @@
       href="./assets/img/icons8-plane-24.png"
     />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./src/styles.scss" />
+    <link href="./dist/styles.css" rel="stylesheet" />
     <title>Flight Track</title>
   </head>
-  <body>
-    <div id="container">
-      <div class="heading">
-        <div class="logo"></div>
-        <div class="flight-track">
-          <p class="flight-heading">FLIGHT</p>
-          <p class="track-heading">TRACK</p>
-        </div>
-      </div>
-      <div class="content">
-        <div class="active-flights">
-          <div class="active-flights-heading">
-            <h1>Active Flights</h1>
-            <span>(Over South-African airspace)</span>
-          </div>
-          <div class="flights">
-            <div class="flight-headings">
-              <span>Call Sign</span>
-              <span class="media full-screen">Vertical Speed</span>
-              <span>Speed</span>
-              <span class="media medium-screen-size">Direction</span>
-              <span>Heading</span>
-              <span class="media large-screen-size">Altitude</span>
-              <span class="flight-location">Flight Location</span>
-            </div>
-            <div id="flights-info"></div>
-          </div>
-        </div>
-        <div class="flight-map">
-          <div id="map"></div>
-        </div>
-      </div>
-    </div>
-    <div id="loading">
-      <div class="spinner">
+  <body
+    class="m-0 h-screen w-screen bg-secondary-bg-color bg-cover font-sans text-white"
+  >
+    <div
+      id="loading"
+      class="absolute top-0 left-0 flex h-full w-full items-center justify-center bg-primary-bg-color flex flex-col"
```

Comment:
it is better to just name the bg stuff as `bg-primary` or `bg-secondary`

Reply:
I have adjusted the code to your recomendation